### PR TITLE
chore: rename delete to remove

### LINKED ISSUE #955: Replace "Delete" account with "Unlink"  account
**Is your feature request related to a problem? Please describe.**
User can remove accounts in Alby when they click on "Delete". However, they might think that they really delete an account and not just unlink it what they actually do. 

**Describe the solution you'd like**
Replace "Delete" account with "Unlink"  account

**Describe alternatives you've considered**
"Remove" Account

**Additional context**
image

**Are you working on this? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Renames delete to remove and updates the alert message.
Follows this https://github.com/getAlby/lightning-browser-extension/issues/955#issuecomment-1153553635

### Link this PR to an issue

Closes #955

### Type of change (Remove other not matching type)

- `chore`: Renaming a variable

### Screenshots of the changes (If any)

Screenshot from 2022-06-16 15-42-15

### How has this been tested?

Ran all tests

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/screens/Accounts/index.tsx [hunk 1]
```diff
@@ -50,14 +50,18 @@ function AccountsScreen() {
   }
 
   async function removeAccount({ id, name }: AccountAction) {
-    if (window.confirm(`Are you sure you want to delete account: ${name}?`)) {
+    if (
+      window.confirm(
+        `Are you sure you want to remove account: ${name}? \nThis can not be undone. If you used this account to login to websites and you might loose access to those.`
```
Comment (bitcoinuser) [inline src/app/screens/Accounts/index.tsx:55]: Needs the (And)?

Comment (im-adithya) [reply src/app/screens/Accounts/index.tsx]: Oops! Fixed it!

**Comment by im-adithya** [comment]
Ready!

**Comment by bitcoinuser** [comment]
I changed my mind. I think it's better use the conceit of Archive account. So you will always be able to undo the action. Same thing of what Blockstream Green wallet does.

What do you think?

**Comment by im-adithya** [comment]
Or else we can combine both of these and could give an option to "archive instead?" below in a modal component instead of a native alert...

**Comment by bitcoinuser** [comment]
Are you sure you want to archive the account: main?
If you used this account to login on sites using Alby, you might not be able to access them.
This action always can be undone.

**Comment by bitcoinuser** [comment]
> Or else we can combine both of these and could give an option to "archive instead?" below in a modal component instead of a native alert... As remove option should always be there for the user at any point of time, wdyt?

I think two options is confusing. Just my opinion.

**Comment by escapedcat** [comment]
Bumi approved this so I think we're good here for now.  
You can always re-add an account. "Archieve" sounds like I can see a list of archived accounts somewhere. Or it would be "greyed out".
Let's go with remove for now. Let's move further discussions to the [design slack channel](https://bitcoindesign.slack.com/archives/C02591ADXM2).

**Pull request merged**